docs/kalman/kalman.md

```diff
@@ -6,204 +6,217 @@ mathjax: true
 author: Ondřej Franek
 ---
 
-# Do I Really Need to Remember All That Stuff Forever? Kalman Filter
+{% raw %}
+<script type="text/x-mathjax-config">
+MathJax.Hub.Config({
+  TeX: {
+    equationNumbers: { autoNumber: "AMS" }
+  }
+});
+</script>
+
+<script type="text/javascript" async
+  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+</script>
+{% endraw %}
+
```

Comment:
```suggestion
```

This should not be a part of this MD file. If needed, change config in https://github.com/ctu-vras/autonomous-robotics/blob/main/_layouts/default.html .